FiveCalls/FiveCalls/IssueContactDetail.swift

```diff
@@ -60,26 +62,29 @@ struct IssueContactDetail: View {
                                 .padding(.leading, 4)
                         }
                     }
-                }
-                .padding(.horizontal)
-                .padding(.bottom)
+                }.padding(.bottom)
                 Text(issue.markdownIssueScript)
-                    .padding(.horizontal)
+                    .padding(.bottom)
                 if remainingContacts.count > 1 {
                     NavigationLink(value: IssueDetailNavModel(issue: issue, contacts: nextContacts)) {
-                        PrimaryButton(title: R.string.localizable.nextContact(),
-                                      systemImageName: "megaphone.fill")
-                        .padding()
+                        OutcomesView(outcomes: issue.outcomeModels, report: { outcome in
+                            let log = ContactLog(issueId: String(issue.id), contactId: currentContact.id, phone: "", outcome: outcome.status, date: Date(), reported: true)
+                            store.dispatch(action: .ReportOutcome(log, outcome))
```

Comment:
At first it seemed a little odd to me to pass a `report` closure into the `OutcomesView`, but I guess it avoids having to expose the issue, currentContact, etc to the sub view.